### DISCUSSION TITLE
Where is the uBO Logger button on Android?

### DISCUSSION THREAD

**Discussion by mtxadmin** [uBlockOrigin/uBlock-issues#2726] [Q&A] [answered]
Some time ago it was on uBO dashboard, but now it was removed? Why?

And the old link for Logger (moz-extension://...) still works, but it is not very user-friendly

**Comment by MasterKia** [comment, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/issues/1817#issuecomment-968969814

https://github.com/uBlockOrigin/uBlock-issues/issues/1817#issuecomment-995115005

**Comment by mtxadmin** [comment]
Thank tou. You are a real magician.
